ipaserver/install/server/install.py

```diff
@@ -760,12 +760,10 @@ def install(installer):
 
     # Create a directory server instance
     if not options.external_cert_files:
-        # Configure ntpd
+        # We have to sync time before generation of certificate and
+        # later ipa-server-uninstall will handle uninstallation
         if not options.no_ntp:
-            ipaclient.install.ntpconf.force_ntpd(sstore)
-            ntp = ntpinstance.NTPInstance(fstore)
-            if not ntp.is_configured():
-                ntp.create_instance()
+            ipaclient.install.client.sync_time(options, fstore, sstore)
```

Comment:
Should we not force `chronyd`?

Reply:
Good catch I will need to discuss this one.